Remotion/ObjectBinding/Web.Development.WebTesting.IntegrationTests/App.config

```diff
@@ -26,6 +26,7 @@
       screenshotDirectory=".\WebTestingOutput"
       logsDirectory=".\WebTestingOutput"
       closeBrowserWindowsOnSetUpAndTearDown="false"
+      cleanUpUnmatchedDownloadedFiles="false"
```

Comment:
See comment in other web test app.config

Reply:
See other comment just an explicit default value to make the patching easier